### PR TITLE
Project template I wrote

### PR DESCRIPTION
**Pull request by Xapollon430**
SUPPP

### CODE REVIEW

**Comment by Xapollon430** [comment]
Last PR

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "codelingual",
+  "version": "1.0.0",
+  "description": "The purpose of this web application is to help individuals quickly learn the basics of new languages by translating standard and common functions from one language to another (so like Google Translate, but for code)!",
```
Comment (candreso) [inline package.json:4]: I'll revise this slightly in an upcoming PR.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "main": "index.js",
+  "scripts": {
+    "server": "nodemon server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Xapollon430/CodeLingual.git"
```
Comment (candreso) [inline package.json:11]: Since my repo is the main one, I'll change the link in an upcoming PR.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Xapollon430/CodeLingual/issues"
```
Comment (candreso) [inline package.json:16]: Note to self: change link.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  "homepage": "https://github.com/Xapollon430/CodeLingual#readme",
```
Comment (candreso) [inline package.json:18]: Change link.

File: models/db.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+env.config();
+
+mongoose.connect(
+  process.env.MONGO_URL,
+  {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  },
+  () => {
+    // We need a db link from the .env file to connec to MongoDB.
```
Comment (candreso) [inline models/db.js:12]: Note to self: fix typo.

File: README.md [hunk 6]
```diff
@@ -1,10 +1,22 @@
...
+
+<h1>HOW TO RUN THE PROJECT</h1>
+Open up 2 terminals either from VS Code or your preffered terminal
+
+Running the server: We will be using nodemon, a library that automatically restarts nodejs code upon saving, on the server.
+Run "npm run server" without the quotes in the top directory where server.js is. Server will run on your http://localhost:5000/
+
+Running the client (React): React directory is in the "client" folder. Our client side will have its own package.json and node/modules just like our server side code will. You need to cd into the client folder and run "npm run start" which will start your react app at http://localhost:3000/
+
+I also made a .gitignore file so we can put folders inside it that we dont want pushed. For example .env, 2 different node modules (our client and server side because these folders are too big to push and pull). This means that our dependencies like react, express, mongoose will not be downloaded when you clone the repository, so you have to run "npm install" in the server directory and client directory to get our dependencies so your code works.
```
Comment (candreso) [inline README.md:21]: NTS: Specify that the "I" is V. This paragraph will probably get removed in the distant future, though.

**Review by candreso — APPROVED** [review]
Merging this PR, as the issues found were minor and mainly documentation related. Thanks for setting this up, @Xapollon430!

**Pull request merged**